### PR TITLE
LA-462,463 Fix bug destination picker

### PR DESCRIPTION
**Pull request by dab246**
**LA-462 Fix bug blank screen when copy files from Workgroup to MySpace in offline mode**

https://user-images.githubusercontent.com/80730648/119472734-afe7eb80-bd74-11eb-8de9-b744db9eeddd.mov

**LA-463 Fix bug pick destination screen is not shown in landscape view**

https://user-images.githubusercontent.com/80730648/119472827-c68e4280-bd74-11eb-87b1-8b764fa9256e.mov

### CODE REVIEW

File: lib/presentation/util/helper/responsive_utils.dart [hunk 1]
```diff
@@ -86,18 +88,44 @@ class ResponsiveUtils {
...
+            horizontal: _destinationPickerHorizontalMarginMediumScreen,
+            vertical: 0.0)
+        : EdgeInsets.symmetric(
+            horizontal: _destinationPickerHorizontalMarginMediumScreen,
+            vertical: _destinationPickerVerticalMarginMediumScreen);
     } else if (isLargeScreen(context)) {
-      return EdgeInsets.symmetric(
-        horizontal: _destinationPickerHorizontalMarginLargeScreen,
-        vertical: _destinationPickerVerticalMarginLargeScreen);
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginLargeScreen * 2
```
Comment (hoangdat) [inline lib/presentation/util/helper/responsive_utils.dart:99]: srr, but what is it `<=`?

File: lib/presentation/util/helper/responsive_utils.dart [hunk 2]
```diff
@@ -86,18 +88,44 @@ class ResponsiveUtils {
...
 
+  BorderRadius getBorderRadiusView(BuildContext context) {
+    if (isMediumScreen(context)) {
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginMediumScreen * 2
+        ? BorderRadius.only(
+            topLeft: Radius.circular(_radiusDestinationPickerView),
+            topRight: Radius.circular(_radiusDestinationPickerView))
+        : BorderRadius.circular(_radiusDestinationPickerView);
+    } else if (isLargeScreen(context)) {
+      return getSizeHeightScreen(context) <= _destinationPickerVerticalMarginLargeScreen * 2
```
Comment (hoangdat) [inline lib/presentation/util/helper/responsive_utils.dart:119]: Can you explain a little bit?

File: lib/presentation/util/helper/responsive_utils.dart (hunk 1 above)
Comment (dab246) [reply lib/presentation/util/helper/responsive_utils.dart:99]: size height screen

File: lib/presentation/util/helper/responsive_utils.dart (hunk 2 above)
Comment (dab246) [reply lib/presentation/util/helper/responsive_utils.dart:119]: We check when the height of the screen is less than or equal to the distance to the margin that we have set when displaying on tablet devices, we will not be separated from the margin vertically but only horizontally.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**